lib/next_ls/test.ex

```diff
@@ -0,0 +1,38 @@
+defmodule NextLS.Test do
```

Comment:
This file should go in the test/support directory. 

Also probably should consolidate with the other test helper module

Reply:
My thought process was other extension devs might use it to test their code actions/workspace commands

Reply:
not a bad idea, but lets cross that road when we get to it